core-services/prow/02_config/openshift-priv/aws-pod-identity-webhook/_pluginconfig.yaml

```diff
@@ -73,6 +73,9 @@ lgtm:
 - repos:
   - openshift-priv/aws-pod-identity-webhook
   review_acts_as_lgtm: true
+- repos:
```

Comment:
@alvaroaleman That doesn't seem right. It seems that we append the same configuration on each autoconfigbrancher PR. This seems like the `private-prow-configs-mirror` tool doesn't delete the original config after the mirror?